docs/workers.md

```diff
@@ -251,6 +257,23 @@ expressions:
     ^/_matrix/client/(api/v1|r0|v3|unstable)/join/
     ^/_matrix/client/(api/v1|r0|v3|unstable)/profile/
 
+    # Typing requests
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/typing
+
+    # Device requests
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/sendToDevice/
+
+    # Account data requests
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/.*/tags
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/.*/account_data
+
+    # Receipts requests
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/receipt
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/read_markers
+
+    # Presence requests
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/presence/.*/status$
+
```

Comment:
Have these endpoints been added to the SyTest worker config so that we actually have tests that these work?

Reply:
Almost all of them were, see matrix-org/sytest#1186.